go.mod

```diff
@@ -0,0 +1,109 @@
+module github.com/influxdata/influxdb
+
+go 1.12
+
+require (
```

Comment:
Did `go mod` pull dependency versions from the old `Gopkg` files? Did it encounter any errors while doing so? Just to make sure we didn't inadvertently change the version of any dependencies.

Reply:
`go mod` copies the dependencies from the `Gopkg.toml` and `Gopkg.lock` files.

the only issues involve warnings about a couple of imports that don't follow the semantic versioning rules:

```
$ go mod init github.com/influxdata/influxdb
go: creating new go.mod: module github.com/influxdata/influxdb
go: copying requirements from Gopkg.lock
go: converting Gopkg.lock: stat github.com/googleapis/gax-go@v2.0.5: github.com/googleapis/gax-go@v2.0.5: invalid version: module contains a go.mod file, so major version must be compatible: should be v0 or v1, not v2
go: converting Gopkg.lock: stat github.com/google/go-github@dd29b543e14c33e6373773f2c5ea008b29aeac95: github.com/google/go-github@v0.0.0-20181009003523-dd29b543e14c: invalid version: go.mod has post-v0 module path "github.com/google/go-github/v18" at revision dd29b543e14c
```

Reply:
I notice in https://github.com/influxdata/influxdb/pull/16903 that @FiloSottile manually added entries for those two. Should this PR do the same?

Reply:
I added them manually and then ran `go mod tidy` which did away with them. Not sure if it impacted the graph though.

Reply:
@FiloSottile thanks. Really appreciate the input on that.